include/types.h

```diff
@@ -4,14 +4,23 @@
 
 typedef uint64_t col_hash_t;
 static const auto& vec_hash = XXH32;
-static const auto& col_hash = XXH64;
+static const auto& col_hash = XXH3_64bits_withSeed;
```

Comment:
I have a faint recollection that doing this may have prevented inlining of XXHash calls. I'll take a look into some assembly to see if this is the case.

I took a look into the assembly and I think we are properly inlining. No changes needed.